features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/org.wso2.carbon.identity.core.server.feature.default.json

```
@@ -2062,6 +2062,7 @@
   "email_templates.branding.fallbacks.button_font_color": "#FFFFFF",
   "email_templates.branding.fallbacks.light_border_color": "transparent",
   "email_templates.branding.fallbacks.dark_border_color": "#333333",
+  "email_templates.branding.fallbacks.support_mail": "support@wso2.com",
```

```suggestion
  "email_templates.branding.fallbacks.support_mail": "",
```
@iff-sal shall we update the fallback value as this ?
Even though we expect our customers to configure a support email in the branding, there can be scenarios where it can be missed. In such cases, users might end up in contacting wso2 support if we have added wso2 support email.

Hence it would be better to set an empty string without explicitly setting wso2 email or a dummy email.